snapshots/terraform/stack/snapshot_scheduler/cloudwatch.tf

```diff
@@ -1,5 +1,6 @@
 resource "aws_cloudwatch_event_rule" "snapshot_scheduler_rule" {
-  name                = "snapshot_scheduler_rule-${var.deployment_service_env}"
-  description         = "Starts the snapshot_scheduler (${var.deployment_service_env}) lambda"
-  schedule_expression = "rate(1 day)"
+  name        = "snapshot_scheduler_rule-${var.deployment_service_env}"
+  description = "Starts the snapshot_scheduler (${var.deployment_service_env}) lambda"
+  # Between 2-3am every day
```

Comment:
?

Reply:
Oops, started of 2-3am but then I realised that might overlap with the Sierra/Calm sync (though that is less of an issue now).